client/analytics/settings/historical-data/status.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Spinner } from '@wordpress/components';
+
+function HistoricalDataStatus( { status } ) {
+	const statusLabels = {
```

Comment:
Nice!  Like the organization here.  I wonder if we should add a filter here to allow third party devs to add in their own import status labels.